app/src/utils/featureFlags.tsx

```diff
@@ -0,0 +1,8 @@
+export function isFeatureFlagOn(featureFlag: string) {
+    console.log(import.meta.env[featureFlag]);
+    return import.meta.env[featureFlag] === 'true';
```

Comment:
just checking -- the default with the flag NOT set anywhere is the current behavior, right?

Reply:
correct, current behavior is the flag not set/set to false!